### PR TITLE
added highlands part 1

### PR DESCRIPTION
**Pull request by burmanp**
May add more later if this gets approved and merged okay.

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -6,5 +6,13 @@
...
-    } 
-]
\ No newline at end of file
+    },
+    {
+        "nameTH": "อุทยานแห่งชาติดอยอินทนนท์",
+        "nameEN": "Doi Inthanon National Park",
+        "lat": 18.59,
+        "long": 98.48,
+        "region": "Thai highlands",
```
Comment (OozyGrub) [inline data.json]:
```suggestion
        "region": "North",
```

**Comment by OozyGrub** [comment]
I'll merge for sure only if issue #7 is completed.

**Comment by burmanp** [comment]
> 

Okay np. I'll add the rest of it after work.

File: data.json [hunk 2]
```diff
@@ -14,5 +14,149 @@
...
+  {
+      "nameTH": "อุทยานแห่งชาติดอยภูนาง",
+      "nameEN": "Doi Phu Nang National Park",
+      "lat": 19.00,
+      "long": 100.16,
+      "region": "Thai highlands",
+      "provinces": ["Phayao"]
+  },
+  {
+      "nameTH": "ตำบล แม่ปืม",
```
Comment (OozyGrub) [inline data.json]:
```suggestion
      "nameTH": "อุทยานแห่งชาติแม่ปืม",
```

File: data.json (hunk 1 above)
Comment (OozyGrub) [reply data.json]: Please change all `Thai highlands` to `North`

**Review by OozyGrub — CHANGES_REQUESTED** [review]

**Comment by OozyGrub** [comment]
@burmanp Thanks for your hard work

**Pull request merged**